.drone.yml

```diff
@@ -344,10 +344,9 @@ steps:
       from_secret: DEBIAN_SECRET_IV
 trigger:
   branch:
-    - master
+    - stable-2.6.0
```

Comment:
So yes, this is stable-2.6.

Reply:
Fixed, and also changed the base branch of the PR to stable-2.6.